js/jsscriptfiles_bubblesort.js

```diff
@@ -0,0 +1,42 @@
+/* This execercise is to sort an array using bubble sort method*/
+
+//The unsorted array
+var unSortedNumberArray = [ 6, 4, 2, 9, 44, 23, 12];
+
+//The variable to initate the functionly only if a number is swapped
+var swapped; 
+
+// Function to sort the array
+function sortAnyArray(takeAnArray){
+	// Condition to check if the array is empty
+	if (takeAnArray.length == 0) {
```

Comment:
Do you know the difference between in == and === in JavaScript?
https://stackoverflow.com/a/359509/3690032

Reply:
Did a quick search, == is comparison and === is literal comparison